docs/reference/commandline/images.md

```diff
@@ -22,25 +22,25 @@ parent = "smn_cli"
       -q, --quiet          Only show numeric IDs
 
 The default `docker images` will show all top level
-images, their repository and tags, and their virtual size.
+images, their repository and tags, and their size.
 
 Docker images have intermediate layers that increase reusability,
 decrease disk usage, and speed up `docker build` by
 allowing each step to be cached. These intermediate layers are not shown
 by default.
 
-The `VIRTUAL SIZE` is the cumulative space taken up by the image and all
+The `SIZE` is the cumulative space taken up by the image and all
```

Comment:
I think "and all its parent images" no longer applies to the new storage model (there's no longer a "parent image"). 

Possibly we can remove this whole paragraph (but I'm okay with dealing with that in a separate PR)